core/src/main/scala/kafka/cluster/Replica.scala

```diff
@@ -130,6 +130,7 @@ class Replica(val brokerId: Int,
     replicaString.append("; Topic: " + partition.topic)
     replicaString.append("; Partition: " + partition.partitionId)
     replicaString.append("; isLocal: " + isLocal)
+    replicaString.append("; lastCUTms: " + _lastCaughtUpTimeMs)
```

Comment:
Hi @ijuma do you prefer the `toString` method to list the `_lastCaughtUpTimeMs` field without abbreviation?